setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="vumi-http-api",
+    version="0.0.1a",
+    url='http://github.com/praekelt/vumi-http-api',
+    license='BSD',
+    description="An HTTP API for interacting with vumi",
+    long_description=open('README.md', 'r').read(),
+    author='Praekelt Foundation',
+    author_email='dev@praekeltfoundation.org',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
+        'vumi',
+        'txJSON-RPC',
```

Comment:
No we don't. I was looking at the wrong thing initially, and forgot to remove this. Will remove.